README.md

```diff
@@ -0,0 +1,2 @@
+Test the action locally:
+```act release -v -W test/test.yml --container-architecture linux/amd64```
```

Comment:
```suggestion
```bash
act release -W test/test.yml --container-architecture linux/amd64```
```
It is easier to copy the command like this.